controllers/app.controllers.js

```diff
@@ -0,0 +1,11 @@
+const { fetchTopics } = require("../models/app.models");
+
+exports.getTopics = (req, res) => {
+  fetchTopics()
+    .then((topics) => {
+      res.status(200).send(topics);
```

Comment:
Anything you send to the body should live free on it but should live in a descriptive key. See sending {topics}

Reply:
Added topics key to object and updated testing suite